### PR TITLE
New version: undergroundwires.privacy.sexy version 0.13.5

### PR DESCRIPTION
**Pull request by averyterrel**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/163052)

### CODE REVIEW

**Comment by averyterrel** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 886.)

File: manifests/u/undergroundwires/privacy/sexy/0.13.5/undergroundwires.privacy.sexy.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: undergroundwires.privacy.sexy
+PackageVersion: 0.13.5
+InstallerType: nullsoft
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/u/undergroundwires/privacy/sexy/0.13.5/undergroundwires.privacy.sexy.installer.yaml:8]:
```suggestion
- Architecture: x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @averyterrel,

Our `x86` validations are performed on 32-bit OSes. 

image

**Comment by stephengillie** [comment]
The [Policy 2.7](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#27-adult-content) check here is due to the "extreme marketing" used in the name and description.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**